console/src/main.tsx

```diff
@@ -9,8 +9,8 @@
 
 import "@/index.css";
 import "@synnaxlabs/media/dist/style.css";
-import "@synnaxlabs/pluto/dist/style.css";
 
+// import "@synnaxlabs/pluto/dist/style.css";
```

Comment:
can this be deleted?

Reply:
No. In fact its critical that this gets commented back in.

Reply:
so why is it not commented in right now